xrpl4j-binary-codec/src/test/java/org/xrpl/xrpl4j/codec/binary/XrplBinaryCodecTest.java

```diff
@@ -120,6 +120,23 @@ void encodeDecodeAmendments() throws JsonProcessingException {
     assertThat(encoder.decode(hex)).isEqualTo(json);
   }
 
+  @Test
+  void encodeUnlModify() throws JsonProcessingException {
+    String json = "{" +
```

Comment:
Can we add another test like this one that validates the `isUNLModify` having the opposite value so we cover both cases?

Reply:
This is covered by tests like `encodeForSigning` and `encodeForMultiSigning` where the txType is Payment and isUNLModify is false.

Reply:
I don't see `UNLModifyDisabling` in those other tests. Is that correct?

Reply:
oh, yes. I just added that! (Sorry, I didn't understand the previous comment correctly.)